cmd/entrypoint/main.go

```diff
@@ -51,6 +52,12 @@ func main() {
 		Waiter:          &realWaiter{},
 		Runner:          &realRunner{},
 		PostWriter:      &realPostWriter{},
+		Results:         strings.Split(*results, ","),
+	}
+	if len(e.Results) >= 1 && e.Results[0] != "" {
```

Comment:
I am curious - what is the purpose of the `e.Results[0] != ""` check?

Reply:
This is because strings.Split("", ",") returns an array with an empty string in it. So an empty string means we try to read the folder as a file and it fails.